src/main/java/seedu/storemando/model/item/ExpiryDateComparator.java

```diff
@@ -0,0 +1,19 @@
+package seedu.storemando.model.item;
+
+import java.util.Comparator;
+
+public class ExpiryDateComparator implements Comparator<Item> {
+
+    @Override
```

Comment:
*Details are in quantityComparator and item class*
Same as the quantityComparator, I think this method body should be abstracted to the item class.

What do others think?

Reply:
I think it is fine to leave it as it is as this is just a custom comparator. Perhaps, the naming of the class can be better.

Reply:
ok will change it